script/DeployConstants.sol

```diff
@@ -0,0 +1,517 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { FinderInterface } from "@uma/core/contracts/data-verification-mechanism/interfaces/FinderInterface.sol";
+import { WETH9Interface } from "../contracts/external/interfaces/WETH9Interface.sol";
+
+/**
+ * @title DeployConstants
+ * @notice Contains constants used in deployment scripts, converted from consts.ts
```

Comment:
Could we read this in via one or more json configuration files? Two advantages:
- Config/code split so it's clear (in the future) which changes are configuration and which changes are logical.
- More flexibility: it will allow us to pull in this info from somewhere else easily. For instance, the json could be generated/updated from an API OR we could import one or more of the config jsons from a common package (like our constants package or something).

Thoughts?

Reply:
Don't they have utilities for that that makes it fairly trivial if the json is well structured? I may be missing some additional complexity (I haven't tried to use these utils before)

https://getfoundry.sh/reference/cheatcodes/parse-json/?highlight=json

Reply:
Thoughts on expanding this, s.t. the Constants.sol file reads in from json as well?